.travis.yml

```diff
@@ -56,16 +67,23 @@ install:
 - source scripts/setup_gae.sh || exit 1
 
 script:
-- if [ $RUN_BACKEND_TESTS == 'true' ] && [ $REPORT_BACKEND_COVERAGE == 'true' ]; then bash scripts/run_backend_tests.sh --generate_coverage_report; fi
-- if [ $RUN_BACKEND_TESTS == 'true' ] && [ $REPORT_BACKEND_COVERAGE == 'false' ]; then bash scripts/run_backend_tests.sh; fi
-- if [ $RUN_LINT == 'true' ]; then bash scripts/install_third_party.sh; python scripts/pre_commit_linter.py --path=.; fi
+- if [ $RUN_E2E_TESTS == 'true' ]; then bash scripts/run_e2e_tests.sh; fi
+- if [ $RUN_BACKEND_TESTS == 'true' ] && [ $REPORT_BACKEND_COVERAGE == 'true' ]; then
```

Comment:
I actually lean here towards keeping each command on one line (as was done before), but if we want to introduce yaml multiline strings, how about going with 'literal' style as described [here](http://stackoverflow.com/a/21699210)? Then you can do proper indentation as well.